src/main/java/screens/View.java

```diff
@@ -222,13 +222,13 @@ private void menuSwitch() {
 
             case QUIZ_MENU:
                 //Menu that displays the quizzes.
-
+                quizScreen.setVisible(true);
 
                 break;
 
             case SHOW_ANSWER:
                 //Menu that shows the answer to a flashcard along with a show next problem button.
-                showAnswerScreen.setQuiz(cardIdArray, currCardIndex, cardAnswer);
+                showAnswerScreen.setQuiz(cardIdArray, currCardIndex, cardAnswer, userAnswer);
```

Comment:
If im understanding this right, both the user answer and card answer should now both be shown on the screen (after a creating a JLabel)?